docs/ros/ros.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: Robot Operating System 2 (ROS 2) Overview
+sidebar_label: ROS 2 Overview
+sidebar_position: 1
+---
+
+For [ROS 2 Humble](https://docs.ros.org/en/humble/index.html) and beyond, the software structure of Clearpath robots has changed significantly. Rather than having individual sets of packages for each
+robot, we have opted to create common `clearpath` packages which are used by all supported platforms. Additionally, we are moving away from using environment variables
+to customize the robot, and instead using a YAML configuration file to describe the robot. We have also defined a [ROS 2 API](api/overview) common to all of our supported platforms. As a result,
+our platforms are now more customizable, more modular, and more unified.
+
+
+## Terminology {#terminology}
+
+***Clearpath Platform***: The *platform* refers to the vehicle that makes up the base of the robot. The platform may be referred to by
+the platform code to indicate the revision.
+
+***Clearpath Robot***: The *robot* refers to the entire robotic system, including the platform, sensors, and additional accessories.
+
+***Clearpath Config***: The *config* refers to the `robot.yaml` configuration file that defines the Clearpath robot.
+
+***Clearpath API***: The *API* refers to the [ROS 2 application programming interface](api/overview). This is a set of ROS 2 topics and services defined
+by Clearpath which are used by all supported Clearpath platforms.
+
+
+## Supported Platforms {#supported-platforms}
+
+<table>
+  <tr>
+    <th>Platform</th>
+    <th>Platform code</th>
+    <th>Supports ROS 2 Humble</th>
+  </tr>
+
+  <tr>
+    <td>Husky</td>
+    <td>A200</td>
+    <td>Yes</td>
+  </tr>
+
+  <tr>
+    <td>Jackal</td>
+    <td>J100</td>
+    <td>Yes</td>
+  </tr>
+
+  <tr>
+    <td>Warthog</td>
+    <td>W100</td>
+    <td>Coming soon</td>
+  </tr>
+
+  <tr>
+    <td rowspan="5">Dingo</td>
+  </tr>
+  <tr>
+    <td>DD100</td>
+    <td>No</td>
+  </tr>
+  <tr>
+    <td>DO100</td>
+    <td>No</td>
+  </tr>
+  <tr>
+    <td>DD150</td>
+    <td>No</td>
+  </tr>
+  <tr>
+    <td>DO150</td>
+    <td>No</td>
+  </tr>
+
+  <tr>
+    <td>Boxer</td>
+    <td>B250</td>
+    <td>No</td>
+  </tr>
+
+  <tr>
+    <td>Ridgeback</td>
+    <td>R100</td>
+    <td>No</td>
+  </tr>
+</table>
+
+## Supported Sensors {#supported-sensors}
+
+<table>
+  <tr>
+    <th>Sensor</th>
+    <th>Type</th>
+    <th>Installation</th>
+    <th>Driver</th>
+  </tr>
+
+  <tr>
+    <td>Hokuyo UST10</td>
```

Comment:
For each of these sensors, can we use the same name as what is used under docs/robots/accessories/sensors? Also, for each of these sensors, can we provide a hyperlink to the corresponding page under accessories?

Reply:
Fixed in https://github.com/clearpathrobotics/cpr-documentation/pull/129